doc/mutations/custom-mutations.md

```diff
@@ -0,0 +1,75 @@
+# The MutationPluginBase plugin
+
+Not all mutations directly related to an entity, and often you might need to perform operations on a mutation that are not necessarily creating, updating or deleting an Entity in Drupal. For these cases you can use the **MutationPluginBase** plugin and extend that instead of extending the **CreateEntityBase** as we saw on *Creating mutations for Entities*.
```

Comment:
Not all mutations are directly related to an entity. 

Write "entity" lowre case.